src/substrait/common/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ add_dependencies(
 target_include_directories(
   substrait_io
   INTERFACE
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../../include/substrait/common>
-    $<INSTALL_INTERFACE:include/substrait/common>)
+    $<BUILD_INTERFACE:${SUBSTRAIT_CPP_INCLUDE_DIR}>
```

Comment:
The linter is complaining here since the lines are so short.  (It wants to use two lines instead of three.)